presto-main/src/test/java/com/facebook/presto/sql/gen/BenchmarkPageProcessor.java

```diff
@@ -108,6 +110,10 @@ public Page handCoded(BenchmarkData data)
         private Tpch1FilterAndProject handcodedProcessor;
         private Page inputPage;
 
+        @SuppressWarnings("unused")
+        @Param({"0.0", "1.0"})
```

Comment:
Current implementation can only support these two values, e.g. no filter and filter that never passes. Hence, it would be clearer to introduce a boolean parameter. 

```
@Param({"true", "false"})
private boolean filterAlwaysFails;
```

Reply:
Changed the callers to the following:

```
        private final RowExpression createFilterExpression(FunctionManager functionManager)
        {
            if (!filterAlwaysFails) {
                return new ConstantExpression(true, BOOLEAN);
            }
            ...
```
and
```
            private boolean filter(int position, Block discountBlock, Block shipDateBlock, Block quantityBlock)
            {
                if (!filterAlwaysFails) {
                    return true;
                }
```